src/components/reference/_triage-states.mdx

```diff
@@ -4,6 +4,10 @@
 | **Ignored** | Findings that are ignored are present in the code, but have been labeled as unimportant. Ignore findings that are false positives or deprioritized issues. Mark findings as ignored through Semgrep Cloud Platform (see [Managing finding status](/semgrep-code/findings/#managing-finding-status) or by adding a code comment (see [nosemgrep comments](/ignoring-files-folders-code/#reference-summary). |
 | **Fixed** | <p>Fixed findings were detected in a previous scan, but are no longer detected in the most recent scan of that same branch. </p> <p> To change the state of a finding to **fixed**: <ol type="a"><li>Fix the code so the rule does not match it.</li><li>Edit the Semgrep rule so it no longer matches the code.</li></ol></p> |
 
+:::info Findings triaged across refs
+Findings triaged (ignored, reopened, resolved) in a specific branch, PR, or MR are also triaged in all other branches, PRs, and MRs of a specific repository. In addition, if you filter for any other [Git references](https://git-scm.com/book/en/v2/Git-Internals-Git-References) (refs) on the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, and then triage a finding, it is also automatically triaged in all other branches, PRs, MRs, and refs.
```

Comment:
@chmccreery does this same thing apply for resolved? I would think if the finding cropped up again after being resolved that it would reappear, but maybe I'm confused.

Reply:
Removed isn't a triage behavior, it just indicates that a finding no longer exists when we scan (e.g. if the rule that found that finding is no longer being used). I don't think we need to specify that here, though.

Reply:
@chmccreery what about resolved findings? I mean - if a code is detected again, it should be detected again - but if a finding is fixed (I should use "fixed" instead of "resolved" to be consistent) - it is only the fixed finding that is triaged.

Therefore, I have to remove "resolved" and just say that this applies to ignored and reopened findings, is this true?

Reply:
Oh yes, that is correct